src/brevitas/core/scaling/pre_scaling.py

```diff
@@ -248,7 +254,7 @@ def __init__(
             restrict_pre_scaling_impl,
             pre_scaling_min_val)
         assert isinstance(
-            pre_zero_point_impl, PreZeroCenterZeroPoint
+            pre_zero_point_impl, get_PreZeroCenterZeroPoint_class()
```

Comment:
Another solution is to check
```suggestion
            pre_zero_point_impl.__class__.__name__, 'PreZeroCenterZeroPoint'
```

I kind of hate the function to import something. I guess this is slightly hard-coded but the check seems fairly hard-coded itself anyway